DiscordBot/commands/tournaments/team/invite.ts

```diff
@@ -0,0 +1,96 @@
+import { ChatInputCommandInteraction, Message, SlashCommandBuilder } from "discord.js";
+import { Command } from "../../";
+import { extractParameters } from "../../../functions/parameterFunctions";
+import commandUser from "../../../functions/commandUser";
+import respond from "../../../functions/respond";
+import { extractTargetText } from "../../../functions/tournamentFunctions/paramaterExtractionFunctions";
+import getTeam from "../../../functions/tournamentFunctions/getTeam";
+import { User } from "../../../../Models/user";
+import getFromList from "../../../functions/getFromList";
+import { invitePlayer } from "../../../../Server/functions/tournaments/teams/inviteFunctions";
+import { loginResponse } from "../../../functions/loginResponse";
+import getUser from "../../../../Server/functions/get/getUser";
+
+async function run (m: Message | ChatInputCommandInteraction) {
+    if (m instanceof ChatInputCommandInteraction)
+        await m.deferReply();
+        
+    const cmdUser = await getUser(commandUser(m).id, "discord", false);
+    if (!cmdUser) {
+        await loginResponse(m);
+        return;
+    }
+
+    const params = extractParameters<parameters>(m, [
+        { name: "team", paramType: "string", optional: true },
+        { name: "user", paramType: "string", customHandler: extractTargetText },
+    ]);
+    if (!params)
+        return;
+
+    const { user, team } = params;
+
+    const tournamentTeam = await getTeam(m, team || cmdUser.ID, team ? "name" : "managerID", cmdUser.ID, true, true);
+    if (!tournamentTeam)
+        return;
+    
+    const users = await User
+        .createQueryBuilder("user")
+        .where("user.osuUsername LIKE :username", { username: user })
+        .orWhere("user.osuUserID = :userID", { userID: user })
+        .getMany();
+
+    const baseUsers = users.map(u => {
+        return {
+            ID: u.ID,
+            name: u.osu.username,
+        };
+    });
```

Comment:
maybe just select their ID and osu! username instead to optimize the query? doesn't look like `users` is used for anything else